recipes/dataframely/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+# Bundle licenses
+cargo-bundle-licenses --format yaml --output ${SRC_DIR}/THIRDPARTY.yml
+
+# Build
+python -m pip install . \
```

Comment:
I expect CARGO_PROFILE_RELEASE_STRIP and CARGO_PROFILE_RELEASE_LTO also do something here: https://github.com/conda-forge/pixi-diff-feedstock/blob/25aa6d0bac276826e7314007a82ad1b0064e6529/recipe/build.sh#L6
(Same for build.bat)